app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @post = Post.find(params[:comment][:post_id])
```

Comment:
- [ ] [OPTIONAL] everything is functioning perfectly 👍. remember, you can still use the strong parameters `comment_params` that are already validated to get the `post_id`.
```suggestion
    @post = Post.find(comment_params[:post_id])
```